src/test/kotlin/it/pagopa/checkout/authservice/controllers/AuthLoginControllerTest.kt

```diff
@@ -0,0 +1,88 @@
+package it.pagopa.checkout.authservice.controllers
+
+import it.pagopa.checkout.authservice.services.AuthLoginService
+import it.pagopa.generated.checkout.authservice.v1.model.LoginResponseDto
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.springframework.http.HttpStatus
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+
+class AuthLoginControllerTest {
+    private lateinit var authLoginController: AuthLoginController
+    private lateinit var authLoginService: AuthLoginService
+
+    @BeforeEach
+    fun setup() {
+        authLoginService = mock()
+        authLoginController = AuthLoginController(authLoginService)
+    }
```

Comment:
```suggestion
    private val authLoginService: AuthLoginService  = mock()

    private val authLoginController= AuthLoginController(authLoginService = authLoginService)
```

no need to instantiate mock and controller before each test, just declare those as private vals

Reply:
Good suggestion, addressed with [5147de5](https://github.com/pagopa/pagopa-checkout-auth-service/pull/12/commits/5147de508232d1618d5ea358037f48e22aeb8778) ✅